dffml/cli/ml.py

```diff
@@ -10,11 +10,15 @@
     KeysCMDConfig,
 )
 from ..base import config, field
+from ..accuracy import AccuracyScorer, MeanSquaredErrorAccuracy
 
 
 @config
 class MLCMDConfig(SourcesCMDConfig, ModelCMDConfig):
-    pass
+    accuracy_scorer: AccuracyScorer = field(
+        "Method to use to score accuracy",
+        default_factory=lambda: MeanSquaredErrorAccuracy(),
```

Comment:
I'm realizing that we should probably not have a default here, because we are not providing a default for the `high_level` functions. This would be an inconsistency.

```suggestion
```